model/functions.php

```diff
@@ -660,4 +660,280 @@ function getAllColumbariumRelatedDataWithFilter(ColumbariumFilter $filter)
         $response -> addError($errorMessage);
     }
     return $response;
+}
+
+function getAllContacts(){
+    $response = new Response();
+    try{
+        $db = connection::getInstance();
+        $con = $db -> get_connection();
+        $query = "select * from contacts;";
+        $statement = $con->prepare($query);       
+        $success = $statement->execute();
+        $result = $statement->fetchAll();
+        $statement->closeCursor();
+        if($success && count($result) > 0)
+        {
+            for($i = 0; $i<count($result); $i++)
+            {
+                $row = $result[$i];
+                $contact = new Contact($row);
+                $response ->addResult($contact);
+            }
+        }
+        if(!$success)
+        {
+            $response -> addError("Failed to fetch Contacts");
+        }
+    }    
+    catch (PDOException $e) {
+        $errorMessage = $e->getMessage();
+        $response -> addError($errorMessage);
+    }
+    return $response;
+}
+
+
+function insertBuriedIndividual(ToBuriedIndividualTable $buriedIndividual)
+{
+    $response = new Response();
+    try{
+        $db = connection::getInstance();
+        $con = $db -> get_connection();
+        $query = "INSERT INTO buried_individuals (FIRST_NAME, MIDDLE_NAME, LAST_NAME, MAIDEN_NAME, DOB, DOD, VETERAN, OBITUARY, TOMB_ID, COLUMBARIUM_ID) 
+            VALUES (:firstName, :middleName, :lastName, :maidenName, :dob, :dod, :veteran, :obituary, :tombId, :columbariumId);";
```

Comment:
Take out `:columbariumId` and `:tombId` this part is only updated from the lot and columbarium requests. We don't have to worry about that here